src/ssmDocument/ssm/ssmServer.ts

```diff
@@ -0,0 +1,503 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { JsonLS, getLanguageServiceSSM as getLanguageService } from 'aws-ssm-document-language-service'
+
+import {
+    createConnection,
+    DidChangeWatchedFilesParams,
+    FileEvent,
+    IConnection,
+    InitializeParams,
+    InitializeResult,
+    NotificationType,
+    RequestType,
+    ServerCapabilities,
+    TextDocuments,
+    TextDocumentSyncKind,
+} from 'vscode-languageserver'
+
+import { posix } from 'path'
+import { clearTimeout, setTimeout } from 'timers'
+import * as URL from 'url'
+import { getLanguageModelCache } from '../../stepFunctions/asl/languageModelCache'
+import { formatError, runSafe, runSafeAsync } from '../../stepFunctions/asl/utils/runner'
```

Comment:
Please discuss with @alanbo if these can be promoted to a more common utility level. Otherwise it may be easy to over look changes made within the stepFunctions code having an impact on ssm docs.

Reply:
Sent an email and still waiting for replies from @alanbo. Are you suggesting something like creating folder under `src/shared` and adding the `languageModelCache` and `utils/runner` files there?